### ISSUE TITLE
new single-page checkout not yet complete

### ISSUE THREAD

**Issue by jflsitecore** [Sitecore/Sitecore.Demo.Retail#54]
We're working on the finishing touches of the new checkout process which is already merged into master.
If you want to complete the checkout process in the meantime please add items to cart and then manually browse to /extendedCheckout instead of /Checkout

Will close this issue once the feature is complete.

**Comment by jflsitecore** [comment]
Fixed by #59
